ubuntu/standard/Dockerfile.template

```diff
@@ -237,7 +237,7 @@ RUN curl https://packages.microsoft.com/config/ubuntu/$(UBUNTU_VERSION)/packages
  && apt-get update \
  && apt-get install -y --no-install-recommends \
     apt-transport-https \
-    dotnet-sdk-2.1 \
+    dotnet-sdk-2.2 \
```

Comment:
Should this be in addition to 2.1? Otherwise this might break for existing users who update the image. Or updating the image is not supposed to be backward compatible?

Reply:
@sachinma According to the MSDN blog post for [2.1](https://blogs.msdn.microsoft.com/dotnet/2018/05/30/announcing-net-core-2-1/)

```.NET Core 2.0 applications will run on .NET Core 2.1 in absence of .NET Core 2.0 being installed. This roll-forward behavior only applies to minor releases. .NET Core 1.1 will not roll-forward to 2.0, nor will .NET Core 2.0 roll-forward to 3.0.```

So if I understand that correctly, all minor versions should continue to work with 2.2. If that's not the case, I can keep 2.1 in addition to 2.2.